src/dev/jest/polyfills/mutation_observer.js

```diff
@@ -0,0 +1,541 @@
+/* eslint-disable */
+// transpiled typescript->javascript from
+// https://github.com/aurelia/pal-nodejs/blob/master/src/polyfills/mutation-observer.ts
+
+/*
+The MIT License (MIT)
```

Comment:
We are going to need a mutation observer polyfill for the browser too no? Is there a reason we need one for jest specifically? Why not use one from npm?

Reply:
A polyfill is not needed for the browser since MutationObservser is implemented in IE 11 and has had support in Firefox, Chrome, and Safari for a long time.

We need a polyfill for jest tests because [jsdom does not implement MutationObservser](https://github.com/jsdom/jsdom/issues/639) and therefore any tests that render a component using MutationObservser will fail to render.

I was using the MutationObservser polyfill that EUI is using. I will see if an npm version exists that we can use.